src/sparkx/Particle.py

```diff
@@ -229,8 +229,8 @@ def __init__(self,input_format=None,particle_array=None):
         if ((input_format is not None) and (particle_array is None)) or ((input_format is None) and (particle_array is not None)):
             raise ValueError("'input_format' or 'particle_array' not given")
         
-        if (particle_array is not None) and not isinstance(particle_array,np.ndarray):
-            raise TypeError("The 'particle_array' is not a numpy array")        
+        #if (particle_array is not None) and not isinstance(particle_array,np.ndarray):
```

Comment:
Why don't you want this check here?

Reply:
I think it should also be allowed to be just a list; do we use the fact that it is a numpy array anywhere? Also the first part was checked before already.

Reply:
Ok, then just delete the test there.